corehq/apps/app_manager/views/modules.py

```diff
@@ -1588,8 +1589,11 @@ def new_module(request, domain, app_id):
                 followup = app.new_form(module_id, _("Followup Form"), lang, attachment=attachment)
                 followup.requires = "case"
                 followup.actions.update_case = UpdateCaseAction(condition=FormActionCondition(type='always'))
-
-            _init_module_case_type(module)
+            case_type = request.POST.get('case_type', None)
+            if case_type:
+                module.case_type = case_type
+            else:
+                _init_module_case_type(module)
```

Comment:
Looking at the definition for `_init_module_case_type`, it defaults to another case type used in the same app. It might be helpful to do something similar in this PR: default the case type dropdown to another case type used in the same module. On the other hand, you're focused on new users, who are less likely to have multiple apps or multiple case types. What do you think?

Reply:
Discussed with Kai, the idea of the feature is to force user think about the case type, so we don't want to provide a default value.

Reply:
cool, thank you for following up